packages/core/admin/admin/src/core/utils/axiosInstance.js

```diff
@@ -33,6 +33,6 @@ instance.interceptors.response.use(
   }
 );
 
-const wrapper = wrapAxiosInstance(instance);
+const wrapper = process.env.NODE_ENV === 'development' ? wrapAxiosInstance(instance) : instance;
```

Comment:
should we just put that in the wrapper code instead ?

Reply:
ok sure